server/chat-plugins/teams.ts

```diff
@@ -0,0 +1,691 @@
+/**
+ * Plugin for sharing / storing teams in a database.
+ * By Mia.
+ * @author mia-pi-git
+ */
+
+import {PostgresDatabase, FS, Utils} from '../../lib';
+import * as crypto from 'crypto';
+
+/** Maximum amount of teams a user can have stored at once. */
+const MAX_TEAMS = 200;
+/** Max teams that can be viewed in a search */
+const MAX_SEARCH = 3000;
+
+export interface StoredTeam {
+	teamid: string;
+	team: string;
+	ownerid: ID;
+	format: ID;
+	title: string | null;
+	date: Date;
+	private: boolean;
```

Comment:
Does the database interface handle conversion from integers to booleans? I'm pretty sure SQLite's `BOOLEAN` type is just an int.

Reply:
We're not using SQLite for this - we're using Postgres.